admin/settings/class-admin-apple-settings-section.php

```diff
@@ -552,6 +557,12 @@ public function render_field( $args ) {
 			$field = '<input required type="text" name="%s" value="%s">';
 		}
 
+		// Add a description, if set.
+		$description = $this->get_description_for( $name );
+		if ( ! empty( $description ) ) {
+			$field .= apply_filters( 'apple_news_field_description_output_html', '<br/><i>' . $description . '</i>', $name );
```

Comment:
It doesn't. Nothing is being output here, so we'd be escaping too early. Also the actual template file has a `wp_kses`. Finally, the `_output_html` in the name is deliberate. This is an obscure VIP convention to indicate a filter that allows filtering of HTML for output and lets them know what the filter is doing.